pkg/controllers/nodeclaim/disruption/drift_test.go

```diff
@@ -21,16 +21,16 @@ import (
 	"knative.dev/pkg/apis"
 	"knative.dev/pkg/ptr"
 
-	"github.com/aws/karpenter-core/pkg/apis/v1beta1"
-	"github.com/aws/karpenter-core/pkg/controllers/nodeclaim/disruption"
-	controllerprov "github.com/aws/karpenter-core/pkg/controllers/nodepool/hash"
-	"github.com/aws/karpenter-core/pkg/operator/controller"
-	"github.com/aws/karpenter-core/pkg/operator/options"
-	. "github.com/aws/karpenter-core/pkg/test/expectations"
+	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	"sigs.k8s.io/karpenter/pkg/controllers/nodeclaim/disruption"
+	controllerprov "sigs.k8s.io/karpenter/pkg/controllers/nodepool/hash"
```

Comment:
While we are changing these paths can we also change this to the correct alias or remove it entirely?